docs/documentation/configuration/parallel.mdx

```diff
@@ -1,34 +1,37 @@
 ---
-title: Search Performance
+title: Read Throughput
 ---
 
 As a general rule of thumb, the performance of expensive search queries can be greatly improved
 if they are able to access more parallel Postgres workers and more shared buffer memory.
 
 ## Parallel Workers
 
-The number of parallel workers depends on the server's CPU count and certain
-Postgres settings in `postgresql.conf`.
+There are three settings that control how many parallel workers ultimately get assigned to a query.
 
-`max_parallel_workers` and `max_worker_processes` control how many workers are available to parallel scans.
-`max_worker_processes` is a global limit for the number of available workers across all connections, and
-`max_parallel_workers` specifies how many of those workers can be used for parallel scans.
+First, `max_worker_processes` is a global limit for the number of workers.
+Next, `max_parallel_workers` is a subset of `max_worker_processes`, and sets the limit for workers used in
+parallel queries. Finally, `max_parallel_workers_per_gather` limits how many workers a _single query_ can receive.
```

Comment:
Nit: I think it's technically "per gather node" ([docs](https://www.postgresql.org/docs/current/runtime-config-resource.html#GUC-MAX-PARALLEL-WORKERS-PER-GATHER)): ... but I've never actually seen a query with more than one gather node, so.